Source/Atomic/Network/Network.cpp

```diff
@@ -596,6 +654,49 @@ void Network::ConfigureNetworkSimulator()
         i->second_->ConfigureNetworkSimulator(simulatedLatency_, simulatedPacketLoss_);
 }
 
+void Network::ClientConnectToMaster(const String& address, unsigned short port)
+{
+    masterServerClient_.ConnectToMaster(address, port);

```

Comment:
I am not sure we should have master server functionality so closely tied to the Network subsystem.  It is possible that a different master server setup would be desired with the Networking system using whatever matchmaking system was in use, for example Steam.  Do you have thoughts on splitting this out?